fedimint-client/src/lib.rs

```diff
@@ -1822,12 +1822,12 @@ impl ClientBuilder {
         Ok(())
     }
 
-    pub fn db(&self) -> &Database {
-        &self.db
+    pub fn db_no_encoders(&self) -> &Database {
```

Comment:
Should be called `db_no_decoders` imo, encoding doesn't require any registry, only decoding.

Reply:
Oops. brainfart.